src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/mapper/CaseDataMapper.java

```diff
@@ -64,6 +65,10 @@ public CaseData buildUpdatedCaseData(CaseData caseData) throws JsonProcessingExc
             .readValue(caseData.getC100RebuildChildDetails(), C100RebuildChildDetailsElements.class);
         updateChildDetailsElementsForCaseData(caseDataBuilder, c100RebuildChildDetailsElements);
 
+        C100RebuildChildDetailsElements c100RebuildOtherChildrenDetailsElements = mapper
```

Comment:
This needs to be revisited, I am not sure if we are using separate field for other children from the frontend

Reply:
changes are done